yb-voyager/cmd/fallForward.go

```diff
@@ -19,8 +19,8 @@ import "github.com/spf13/cobra"
 
 var fallForwardCmd = &cobra.Command{
 	Use:   "fall-forward",
-	Short: "fall-forward is used to setup and synchronize the fall forward database",
-	Long:  `Fall-forward has three commands: setup, synchronize and switchover.`,
+	Short: "Manage the fall-forward backup strategy for live migration by setting up a source-replica DB and synchronizing it with the source/target",
```

Comment:
why have we changed the terminology of fall-forward database to source-replica db?

Reply:
Ah sorry, made this change by mistake. Wanted to discuss this first as part of UX changes